Tasks/VsTest/vstest.ts

```diff
@@ -908,34 +955,48 @@ function pushImpactLevelAndRootPathIfNotFound(dataCollectorArray): void {
     }
 }
 
+function roothPathGenerator() : any
+{
```

Comment:
Rest of the file and all other tasks have K&R bracing.  fix in master.  be consistent